src/test/operators/aggregate_test.cpp

```diff
@@ -472,6 +473,34 @@ TEST_F(OperatorsAggregateTest, DictionarySingleAggregateCountWithNull) {
                     "src/test/tables/aggregateoperator/groupby_int_1gb_1agg/count_null.tbl", 1, false);
 }
 
+/**
+ * Tests for empty tables
+ */
+
+TEST_F(OperatorsAggregateTest, TwoAggregateEmptyTable) {
+  auto filtered = std::make_shared<TableScan>(_table_wrapper_1_2, ColumnID{0}, PredicateCondition::LessThan, 0);
+  filtered->execute();
+  this->test_output(filtered,
+                    {{ColumnID{1}, AggregateFunction::Max},
+                     {ColumnID{2}, AggregateFunction::Count},
+                     {std::nullopt, AggregateFunction::Count}},
+                    {}, "src/test/tables/aggregateoperator/0gb_3agg/max_count_count_empty.tbl", 1);
+}
+
+TEST_F(OperatorsAggregateTest, TwoAggregateEmptyTableGrouped) {
+  auto filtered = std::make_shared<TableScan>(_table_wrapper_1_2, ColumnID{0}, PredicateCondition::LessThan, 0);
+  filtered->execute();
+  this->test_output(filtered,
+                    {{ColumnID{1}, AggregateFunction::Max},
+                     {ColumnID{2}, AggregateFunction::Count},
+                     {std::nullopt, AggregateFunction::Count}},
+                    {ColumnID{0}}, "src/test/tables/aggregateoperator/groupby_int_1gb_3agg/max_count_count_empty.tbl",
+                    1);
+}
+
+// (debug)> SELECT COUNT(a) FROM a WHERE a>1000000
```

Comment:
what are these lines for?

Reply:
what lines? ![hmm](http://forum.mods.de/bb/img/smilies/icon16.gif)